modules/distr-tracing-tempo-install-web-console.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-web-console_{context}"]
+= Installing the {TempoShortName} by using the OpenShift web console
+
+You can install the {TempoShortName} in the OpenShift web console.
+
+.Prerequisites
+
+* You are logged in to the OpenShift web console as a cluster administrator with the `cluster-admin` role.
+
+* You are using a supported provider of object storage: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+.Procedure
```

Comment:
I agree that second-level ordered list items and the other details that you've mentioned are not ideal, but the alternative of breaking it into multiple procedures also has its drawbacks, including readability drawbacks (like having to complete several procedures instead of one).